conda_verify/utilities.py

```diff
@@ -113,6 +114,7 @@ def get_object_type(data):
         pos = data.find(b'PE\0\0')
         if pos < 0:
             return "<no PE header found>"
+        data = array.array('B', data)
```

Comment:
We can remove this line and add ``data = future.builtins.bytes(data)`` on the line after the function header.